src/content/structured/components/divider/accessibility.mdx

```diff
@@ -0,0 +1,49 @@
+---
+path: "/components/divider/accessibility"
+
+date: "2024-12-02"
+
+title: "Divider"
+
+status: "BETA"
+
+subTitle: "Dividers are used to separate different elements in groups of content. Visually, they look like rules or lines."
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/divider" },
+    { title: "Code", path: "/components/divider/code" },
+    { title: "Accessibility", path: "/components/divider/accessibility" },
+  ]
+---
+
+## Easy to use for everyone
+
+Horizontal dividers use the semantic `<hr>` ('thematic break' or 'horizontal rule') HTML element.
+
+Vertical dividers use the `separator` role and have an `aria-orientation` attribute set to `vertical`. These attributes are implicitly present on horizontal dividers (but with an `aria-orientation` of `horizontal`) because of the use of the `<hr>` element.
```

Comment:
```suggestion
Vertical dividers use the `separator` role and have an `aria-orientation` attribute set to `vertical`. Horizontal dividers with a label use the `separator` role and have an `aria-orientation` attribute set to `horizontal`. These attributes are implicitly present on horizontal dividers with no label (but with an `aria-orientation` of `horizontal`) because of the use of the `<hr>` element.
```